tgui/packages/common/config_setter.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * This is a pair file with `tgui_config.js` in config directory
+ * @file
+ * @copyright 2023 EvilDragonfiend
+ * @license MIT
+ */
+
+import { PREF_ADDITION_KEY } from '../../../config/tgui_config';
```

Comment:
This will cause issues on already existing config folders by throwing an error when deployed by causing a file not found error

Reply:
addressed

Reply:
how?

Reply:
Also for TGS tgui is compiled before symlinks are brought over so this wouldn't work. config/ the folder is presumed to not be needed for compilation and is only around at runtime.